### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by verissimohenry**

### CODE REVIEW

**Comment by wathigo** [comment]
> # Invalid PR
The pull request has been marked as invalid since linter checks are failing. 
Let's ensure linters checks are passing.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

Hi, there I have noticed that you have skipped several requirements from the [previous TSE](https://github.com/verissimohenry/newyorktime/pull/3#pullrequestreview-592475410) Please return back and make sure you do them, also I have left for you some of the changes to do.

Please check the comments below.
:point_down: :point_down: :point_down: :point_down: :point_down: 

## Require Changes :red_circle::

- [ ] Please use the same colors as in the original page as well as the layout and position elements correctly.

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes Required :yellow_circle: 
### Hello  @verissimohenry :wave: 

### Thanks for working on the changes suggested and your project passes most of the requirements, but you still can improve your project. 
#### Please work on all requirements suggested by previous TSEs and I have replicated the remaining requirements below so that you can refer them easily.

- [ ] First of all, always provide a description of the features you have added in your pull request. Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

- [ ] Please write a descriptive and well-structured README file for your project (Also give yourself a quick reminder from this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/))

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Comment by verissimohenry** [comment]
Add a well-descriptive README file to my project.

File: index.html [hunk 1]
```diff
@@ -2,786 +2,838 @@
...
+                    the<br />
+                    Center for Astrophysics here on Monday and in a set of papers<br />
+                    submitted to The Astrophysical Journal.
+                </p>
+            </div>
 
+        </div>
+        <br>
+        <section>
+            <h2>The Theory of Inflation</h2>
```
Comment (Forison) [inline index.html]: moving 195 to 200 down

File: index.html [hunk 2]
```diff
@@ -2,786 +2,838 @@
...
+            <p class="summary-text">
+                <span class="summary-text">Astronomers have found evidence to support the theory of inflation,
+                    which explains how the universe expanded so uniformly and so quickly
+                    in the instant after the Big Bang 13.8 billion years ago.</span>
+            </p>
             <section class="graphics">
```
Comment (Forison) [inline index.html:200]: to having 

`

            <section class="graphics">
              <h2>The Theory of Inflation</h2>
              <p class="summary-text">
                  <span class="summary-text">Astronomers have found evidence to support the theory of inflation,
                      which explains how the universe expanded so uniformly and so quickly
                      in the instant after the Big Bang 13.8 billion years ago.</span>
              </p>
<h2 hidden></h2>
                <div class="img-tool">
                    first image
                    <p class="img-p">
                        <strong>THE UNIVERSE</strong> is just under 14 billion years old.
                        From our position in the Milky Way galaxy, we can observe a sphere
                        that is now about 92 billion light-years across. But there's a
                        mystery. Wherever we look, the universe has an even temperature.
                    </p>
                </div>
.....
`
will fix the theory of inflation UI flaw.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change Required🛑

# 🙋‍♂️ Team 

* [ ] As mentioned in the previous review, you need to add a descriptive Readme for the page, Let us use the format, given in the [pathright by Microverse](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).

* [ ] Some portions of the page do not look like  the original :

####  Kindly align the items in navbar to the center.

- There should be some margin at the top of the button.



- There should be some margin at the top to make it align to the center of the page.

Screenshot from 2021-02-21 13-58-48

You can use the `align-items-center` attribute on the parent navbar to have element aligned to the center

#### Let us take a second look at this part of the page 
cloned page
Screenshot from 2021-02-21 14-03-41

let us compare it to the original 
Screenshot from 2021-02-21 14-05-35

Let us check the test under the image, kindly restructure the code to get the right alignment.

#### Let us take a look at theory of inflation section

original

Screenshot from 2021-02-21 14-11-55

clone

Screenshot from 2021-02-21 14-15-47
 check comment below

#### That is it, kindly make the changes and submit a re-review👷.

#### Happy coding 👨‍💻

Addo Forison

**Comment by verissimohenry** [comment]
Done.

**Pull request closed without merging**